docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -24,7 +24,25 @@ The variables used are different for each supported cloud provider.
 
 ### Azure
 
-To discover Azure cloud resources, Octopus uses an [Azure account](/docs/projects/variables/azure-account-variables.md) variable named **Octopus.Azure.Account**.
+To discover Azure cloud resources, Octopus uses the following variables:
+
+| Name                    | Required | Description                                                                                                   |
+| ----------------------- | -------- | ------------------------------------------------------------------------------------------------------------- |
+| `Octopus.Azure.Account` | Y        | An [Azure account](/docs/projects/variables/azure-account-variables.md) to use when discovering cloud targets |
+
+### AWS
+
+To discover AWS cloud resources, Octopus uses the following variables:
+
+| Name                                      | Required | Description                                                                                                                                                                                                                                                  |
+| ----------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `Octopus.Aws.Regions`                     | Y        | A comma separated list of AWS regions to perform target discovery in.                                                                                                                                                                                        |
+| `Octopus.Aws.Account`                     | N        | An [AWS account](/docs/projects/variables/aws-account-variables.md) account to use when discovering cloud targets. If this is not set then credentials from the worker on which the step is run will be used.                                                |
+| `Octopus.Aws.WorkerPool`                  | N        | A [worker pool](/docs/projects/variables/worker-pool-variables.md) to use when discovering cloud targets. If this is not set then the worker pool from the step will be used. If this is set any discovered targets will have this set as their worker pool. |
```

Comment:
Worker Credentials are only used if the `Octopus.Aws.WorkerPool` variable is set - otherwise it will just not do discovery if no `Octopus.Aws.Account` is set.

Reply:
That behaviour was recently changed under [this PR](https://github.com/OctopusDeploy/OctopusDeploy/pull/12477) so that target discovery is gated by the `Octopus.Aws.Regions` variable

Reply:
Ah apologies, I thought that was still on the todo list 👍